### ISSUE TITLE
Switch to fullcalendar module

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9790]
# Motivation

Siteimprove is reporting compliance issues with the calendar display in our current implementation using fullcalendar_views. This version is also not compatible with D11.

In https://github.com/uiowa/uiowa/issues/9365, we were able to deprecate a couple usages of this across sites and came up with a recommendation based off our investigation (https://github.com/uiowa/uiowa/issues/9365#issuecomment-4337215817) to move to https://www.drupal.org/project/fullcalendar which still doesn't meet accessibility compliance but positions us better for D11 and additional future accessibility enhancements.

Assuming there are still accessibility issues after we make this switch, we will want a follow up issue to provide an alternate accessible list display that meets compliance or further upgrades depending on the accessibility of future module releases.

# Additional Info

Investigation and recommendation - https://github.com/uiowa/uiowa/issues/9365#issuecomment-4337215817

Existing PR - https://github.com/uiowa/uiowa/pull/9785

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Complicated

 **Discussion:**
 - styling changes will be needed based on different class names
   - span floats vs flexbox
 - https://github.com/uiowa/uiowa/pull/9785
   - only targets its alerts calendar
   - event split shows config that is using fullcalendar_views
   - current pr disables list display. We should enable that again until we address accessibility more in a future slice
 - May have additional styles we introduced to the fullcalendar_views output (sticky header)
   - sticky header may be culprit for one of the existing accessibility issues

**Proposed solution or definition of done:** 
- Update all instances of fullcalendar_views to use fullcalendar module
- enable the list display that has been disabled in the existing PR
- address any styling needs
- see about resolving accessibility issue related to sticky header (Scrollable element is not keyboard accesible)

**Comment by joewhitsitt** [comment]
Ope. Did this not close with the merge? Anything else here?

**Comment by briand44** [comment]
I think it can close. I don't think we resolved "see about resolving accessibility issue related to sticky header (Scrollable element is not keyboard accesible)" but I can create a follow up slice to address remaining accessibility issues prior to new compliance deadline.
